Firestore/core/src/firebase/firestore/model/field_value.h

```diff
@@ -153,6 +146,12 @@ class FieldValue : public util::Comparable<FieldValue> {
     return *blob_value_;
   }
 
+  /**
+   * Returns a string_view of the blob_value(). This can be useful when using
+   * abseil bytewise APIs that accept this type.
+   */
+  absl::string_view blob_value_as_string_view() const;
```

Comment:
Very optional: in a similar vein, perhaps `blob_value/blob_size` could be replaced by a single function that returns an `absl::span`?